CMakeLists.txt

```diff
@@ -103,7 +103,22 @@ tristate_option(WITH_USDT
 )
 cmake_dependent_option(WITH_EXTERNAL_SIGNER "Enable external signer support." ON "NOT WIN32" OFF)
 set(ENABLE_EXTERNAL_SIGNER ${WITH_EXTERNAL_SIGNER})
-tristate_option(WITH_QRENCODE "Enable QR code support." "if libqrencode is found." AUTO)
+
+include(cmake/optional_qt.cmake)
+cmake_dependent_option(WITH_QRENCODE "Enable QR code support." ON "WITH_GUI" OFF)
+if(WITH_QRENCODE)
+  if(VCPKG_TARGET_TRIPLET)
+    # TODO: vcpkg fails to build libqrencode package due to
```

Comment:
I see this is move only, but we shouldn't be adding comments like this to our build system, as we aren't a vcpkg (or any other package manager) issue tracker. Especially since this is about a (sub)-dependency we don't even use.

Reply:
I agree. The comment has been dropped.